src/components/FormControlWrapper.tsx

```diff
@@ -27,3 +27,28 @@ export function FormControlWrapper(
     />
   );
 }
+
+export function InputElementWrapper(
+  props: React.InputHTMLAttributes<HTMLInputElement>
+): JSX.Element {
+  const { setIsInputElementInFocus } = useInputElementFocus();
```

Comment:
what is this? and what is this FocusContextProvider for?

Reply:
the FocusContextProvider existed and provides focused elements for which a press of `/` does (not) invoke a search

Reply:
ah true, forgot about that